frameworks/PHP/fat-free/deploy/nginx.conf

```diff
@@ -42,7 +45,7 @@ http {
 
     upstream fastcgi_backend {
         server unix:/var/run/php/php7.3-fpm.sock;
-        keepalive 40;
+        keepalive 120;
```

Comment:
That change will be detrimental.

It's the number of `keealive` connections to the `php-fpm`, per worker processes (1 per cpu core).

28 cores * 120 = 3,360
`php-fpm` only have 1.024 children processes.

Reply:
@joanhey That could be true. I was testing it on my machine and it may be biased towards my machine than on another. I just noticed performance increased when I bumped this number up (but too much and performance sank again).